app/views/guitars/delete.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Guitars#delete</h1>
```

Comment:
Since we will be using the CRUD actions as api only, the template files for them can be deleted

Reply:
OK I'll remove them 👍🏼